aws/table_aws_ses_email_identity.go

```diff
@@ -0,0 +1,180 @@
+				Description: "The ARN of the AWS SES identity.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEmailIdentityARN,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEmailIdentityAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listSESEmailIdentities(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("listSESEmailIdentities")
+
+	region := d.KeyColumnQualString(matrixKeyRegion)
+
+	// Create Session
+	svc, err := SESService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// execute list call
+	input := &ses.ListIdentitiesInput{IdentityType: &ses.IdentityType_Values()[0]}
+	identityResult, err := svc.ListIdentities(input)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, identity := range identityResult.Identities {
+		d.StreamListItem(ctx, *identity)
+
+		// Context may get cancelled due to manual cancellation or if the limit has been reached
+		if d.QueryStatus.RowsRemaining(ctx) == 0 {
+			return nil, nil
+		}
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getEmailIdentityVerificationAttributes(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getEmailIdentityVerificationAttributes")
+
+	name := h.Item.(string)
+	region := d.KeyColumnQualString(matrixKeyRegion)
+	identities := []*string{&name}
+
+	// Create Session
+	svc, err := SESService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &ses.GetIdentityVerificationAttributesInput{
+		Identities: identities,
+	}
+	result, err := svc.GetIdentityVerificationAttributes(input)
+	if err != nil {
+		return nil, err
+	}
+	return result.VerificationAttributes[name], err
+}
+
+func getEmailIdentityNotificationAttributes(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getEmailIdentityNotificationAttributes")
+
+	name := h.Item.(string)
+	region := d.KeyColumnQualString(matrixKeyRegion)
+	identities := []*string{&name}
+
+	// Create Session
+	svc, err := SESService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &ses.GetIdentityNotificationAttributesInput{
+		Identities: identities,
+	}
+	result, err := svc.GetIdentityNotificationAttributes(input)
+	if err != nil {
+		return nil, err
+	}
+	return result.NotificationAttributes[name], err
+}
+
+//// TRANSFORM FUNCTIONS
+
+func getEmailIdentityARN(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
```

Comment:
Transform function should not take `*plugin.HydrateData` as the parameter, we should use `*transform.TransformData` instead.